packages/example/src/containers/Dashboard/Dashboard.tsx

```diff
@@ -7,12 +7,23 @@ import {Account} from "../../components/Account/Account";
 import {MetaMaskConnector} from "../MetaMaskConnector/MetaMaskConnector";
 import {MetaMaskContext} from "../../context/metamask";
 import {LatestBlock} from "../../components/LatestBlock/LatestBlock";
-import {getAddress, getBalance, getPublicKey} from "../../services/account";
-import {getLatestBlock} from "../../services/block";
-import {addPolkadotAsset} from "../../services/asset";
-import {setConfiguration} from "../../services/configuration";
+// import {getAddress, getBalance, getPublicKey} from "../../services/account";
+// import {getLatestBlock} from "../../services/block";
+// import {addPolkadotAsset} from "../../services/asset";
+// import {setConfiguration} from "../../services/configuration";
 import {getPolkadotApi} from "../../services/polkadot";
 import {BlockInfo} from "@nodefactory/metamask-polkadot-types";
+import {web3Enable, web3Accounts, web3ListRpcProviders, web3FromAddress} from "@polkadot/extension-dapp";
+import {InjectedExtension} from "@polkadot/extension-inject/types"
+import {injectMetamaskPolkadotSnapProvider} from "@nodefactory/metamask-polkadot-adapter/";
+import {MetamaskSnapApi} from "@nodefactory/metamask-polkadot-adapter/src/types";
+
+export interface IPolkadotAdapter extends InjectedExtension {
+    getMetamaskSnapApi: () => Promise<MetamaskSnapApi>;
+    enableSnap: () => void;
+    pluginOrigin: string;
+    config: {"networkname" : string};
```

Comment:
you don't need those:
enableSnap: () => void;
    pluginOrigin: string;
    config: {"networkname" : string};